AirportTicketNookingTest/FliterBookingShould.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Airport_Ticket_Booking;
+using Xunit;
+using static Airport_Ticket_Booking.Common;
+
+namespace AirportTicketNookingTest
+{
+    public class FliterBookingShould
```

Comment:
can you search for how to build tear-down methods, these methods are used to clear the state between tests. we can discuss that.